### DISCUSSION TITLE
主页构建器V1

### DISCUSSION THREAD

**Discussion by MFn233** [MFn233/PCL-Mainpage-Market#7] [Announcements]
这是主页构建器的第一个release。

in.txt中，包括了用自定义文本格式书写的文本，编码为UTF-8

## 例子

```
卡片1
https://example.org/
::disc
简介第一行
简介第二行
::enddisc
卡片2
https://example.org/
::disc
简介第一行
简介第二行
简介第三行
::enddisc
::exit
```

期中，"::disc"与"::enddisc"分别代表简介开始与结束，

"::exit"代表文件结束

所有文本需要严格按照上面的形式编写，不能出现空行

## 如何build？

1. 打开“构建器源代码”根目录

2. 确保你已正确安装MinGW64，并已配置环境变量

3. 控制台输入:
> g++ Constructor.cpp -o Constructor.cpp

## 投稿主页

请发送label为“新主页”的issue，以方便筛选与查找

完整包括**主页名、简介、直链**，其中直链可以是指向xaml的链接，也可以是主页介绍页面之类的东西

最好用自定义文本格式在monospace中写一遍，当然你也可以不这么做，我们会完成这一步骤。

**Full Changelog**: https://github.com/MFn233/PCL-Mainpage-Market/commits/v1

<hr /><em>This discussion was created from the release <a href='https://github.com/MFn233/PCL-Mainpage-Market/releases/tag/Constructor_v1'>主页构建器V1</a>.</em>

**Comment by Ad-closeNN** [comment]
~~README还是晚了一步~~
![image](https://github.com/MFn233/PCL-Mainpage-Market/assets/103355026/045c673a-8ed5-4a71-a7f6-a2692dcb33e5)

**Reply by SYBass** [reply]
因为我和MFn讨论了一下发现还是向大家推荐主页是最好

**Comment by Ad-closeNN** [comment]
其实现在还没有名为“新主页”的label（
sybass如果在的话麻烦加个label新主页呗
到时候直接用这个就能提交issues
[https://github.com/MFn233/PCL-Mainpage-Market/issues/new?labels=新主页&title=[新主页] ](https://github.com/MFn233/PCL-Mainpage-Market/issues/new?labels=新主页&title=[新主页] )

#### PCL issue的测试
[https://github.com/Hex-Dragon/PCL2/issues/new?labels=新反馈&title=龙猫啊啊啊啊&projects=&template=bug9.yml](https://github.com/Hex-Dragon/PCL2/issues/new?labels=新反馈&title=龙猫啊啊啊啊&projects=&template=bug9.yml)

**Reply by SYBass** [reply]
有的哦~

**Reply by Ad-closeNN** [reply]
无法自行选择 :(
~~有但用不了~~

**Comment by SqrtTan** [comment]
等会，这玩意我看懂了，但信标的构建器我到现在也没搞懂咋工作的
